src/test/java/core/basesyntax/BalanceOperationTest.java

```diff
@@ -0,0 +1,86 @@
+package core.basesyntax;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.ServiceShopForTestsException;
+import core.basesyntax.service.operation.BalanceOperation;
+import core.basesyntax.service.operation.OperationHandler;
+import core.basesyntax.service.operation.ServiceShopForTests;
+import core.basesyntax.service.operation.ServiceShopForTestsImpl;
+import core.basesyntax.service.operation.SupplyOperation;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Test;
+
+public class BalanceOperationTest {
+    private ServiceShopForTests serviceShopForTests = new ServiceShopForTestsImpl();
```

Comment:
remove this usage of ServiceShopForTests class. You should test BalanceOperation class. Not ServiceShopForTests